example/cumulus-tf/browse_example.tf

```diff
@@ -19,6 +19,7 @@ module "discover_granules_browse_example_workflow" {
           "event.$": "$",
           "task_config": {
             "provider": "{$.meta.provider}",
+            "provider_path": "{$.meta.collection.provider_path}",
```

Comment:
Do we want to also update the ingest browse data cookbook to specify that the rule should have the provider path?

https://nasa.github.io/cumulus/docs/data-cookbooks/browse-generation

Reply:
Done: caa432feb

Reply:
I'm wondering if it should also be updated here: https://nasa.github.io/cumulus/docs/data-cookbooks/run-tasks-in-lambda-or-docker

The purpose of that doc isn't necessarily to show a "working" discover granules workflow, but people might easily copy/paste from it and be confused when it doesn't work

Reply:
Fixed here: 1b5706565